app/(tabs)/_layout.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Tabs } from 'expo-router';
+import {
+  Image,
+  ImageBackground,
+  ImageSourcePropType,
+  View,
+} from 'react-native';
+import { images } from '@/constants/images';
+import { Text } from 'react-native';
+import { icons } from '@/constants/icons';
+
+type TabIconProps = {
+  focused: boolean;
+  title: string;
+  icon: ImageSourcePropType;
+};
+const TabIcon = ({ focused, title, icon }: TabIconProps) => {
+  if (focused) {
+    return (
+      <ImageBackground
+        source={images.highlight}
+        className="flex flex-row w-ful flex-1 min-w-28 min-h-16 mt-4 justify-center items-center rounded-full overflow-hidden"
+      >
+        <Image source={icon} tintColor="#151312" className="size-5" />
+        <Text className="text-secundary text-base font-semibold ml-2">
+          {title}
+        </Text>
+      </ImageBackground>
+    );
+  }
+  return (
+    <View className="size-full justify-center items-center mt-4 rounded-full">
+      <Image source={icon} tintColor="#A8B5DB" className="size-5" />
+    </View>
+  );
+};
+
+const _layout = () => {
```

Comment:
Is this the usual name for this? Or a react native best practice? what would you suggest?